homeassistant/components/met_eireann/config_flow.py

```diff
@@ -0,0 +1,77 @@
+"""Config flow to configure Met component."""
+from typing import Any, Dict, Optional
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_ELEVATION, CONF_LATITUDE, CONF_LONGITUDE, CONF_NAME
+from homeassistant.core import callback
+import homeassistant.helpers.config_validation as cv
+
+from .const import DOMAIN, HOME_LOCATION_NAME
+
+
+@callback
+def configured_instances(hass):
+    """Return a set of configured SimpliSafe instances."""
+    entries = []
+    for entry in hass.config_entries.async_entries(DOMAIN):
+        entries.append(
+            f"{entry.data.get(CONF_LATITUDE)}-{entry.data.get(CONF_LONGITUDE)}"
+        )
+    return set(entries)
+
+
+class MetEireannFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
+    """Config flow for Met Eireann component."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
+
+    def __init__(self):
+        """Init MetEireannFlowHandler."""
+        self._errors = {}
+
+    async def async_step_user(self, user_input=None):
+        """Handle a flow initialized by the user."""
+        self._errors = {}
+
+        if user_input is not None:
+            if (
+                f"{user_input.get(CONF_LATITUDE)}-{user_input.get(CONF_LONGITUDE)}"
```

Comment:
I believe you can use this instead:

https://developers.home-assistant.io/docs/config_entries_config_flow_handler#unique-ids